public/js/graph/react_graph.js

```diff
@@ -438,6 +439,39 @@ var Graph = React.createClass({
         });
     },
 
+    // Zoom and pan function calls work once, but they are not repeated while mouse is down, as wanted.
+    onButtonPress: function(zoomOrPanFunction, direction) {
+        this.setState({mouseDown: setInterval(zoomOrPanFunction(direction), 50)});
```

Comment:
You need to pass in a function, not a function call (which doesn't return anything in this case). So probably use an anonymous function.